### PR TITLE
Update documenation for ExperimentalDesign

### PR DESCRIPTION
**Pull request by timosachsenberg**
# Description

Fix for docu.

Also please:
- Make sure that you are listed in the AUTHORS file
- Add relevant changes and new features to the CHANGELOG file

# Checklist:
- [X] I have commented my code, particularly in hard-to-understand areas
- [X] New and existing unit tests pass locally with my changes
- [X] Updated or added python bindings for changed or new classes

### CODE REVIEW

File: src/openms/include/OpenMS/METADATA/ExperimentalDesign.h [hunk 1]
```diff
@@ -62,21 +63,26 @@ namespace OpenMS
...
-    MSstats_Condition: a string that indicates the condition (e.g. control or 1000 mMol). Will be forwarded to MSstats and 
+    - MSstats_Condition: a string that indicates the condition (e.g. control or 1000 mMol). Will be forwarded to MSstats and 
                        can then be used to specify test contrasts.
-    MSstats_BioReplicate: a numeric identifier to indicate replication. MSstats requires that there are no duplicate entries. 
+
+    - MSstats_BioReplicate: a numeric identifier to indicate replication. MSstats requires that there are no duplicate entries. 
                           E.g., if MSstats_Condition, Fraction_Group group, and Fraction number are the same - 
                           as in the case of biological or technical replication, 
                           one uses the MSstats_BioReplicate to make entries non-unique)
+
```
Comment (jpfeuffer) [inline src/openms/include/OpenMS/METADATA/ExperimentalDesign.h:85]: MSstats_mixture is missing

File: src/openms/include/OpenMS/FORMAT/ExperimentalDesignFile.h [hunk 2]
```diff
@@ -43,64 +43,7 @@ namespace OpenMS
...
-	Sample	Some_Condition1	Some_Condition2
-	1	1	1
-	2	2	1
-	3	3	1
-	4	4	1
-	5	1	2
-	6	2	2
-	7	3	2
-	8	4	2
+  @brief Load an ExperimentalDesign from a TSV file.
```
Comment (jpfeuffer) [inline src/openms/include/OpenMS/FORMAT/ExperimentalDesignFile.h:46]: Can you at least leave a reference to the underlying class which describes the content of the file?

**Review by jpfeuffer — CHANGES_REQUESTED** [review]

File: src/openms/include/OpenMS/FORMAT/ExperimentalDesignFile.h (hunk 2 above)
Comment (timosachsenberg) [reply src/openms/include/OpenMS/FORMAT/ExperimentalDesignFile.h:46]: doxygen adds a link to ExperimentalDesign automatically (it recognizes class names).
Is this sufficient?

File: src/openms/include/OpenMS/METADATA/ExperimentalDesign.h (hunk 1 above)
Comment (timosachsenberg) [reply src/openms/include/OpenMS/METADATA/ExperimentalDesign.h:85]: Ok good point. Will add to it.

**Review by jpfeuffer — APPROVED** [review]

**Pull request merged**